internals/build.rs

```diff
@@ -21,10 +24,37 @@ fn main() {
     let minor = version_components
         .next()
         .unwrap_or("0")
-        .parse::<u64>()
+        .parse::<usize>()
         .expect("invalid Rust minor version");
 
-    for version in &[53, 55, 60] {
+    let manifest_dir = std::env::var_os("CARGO_MANIFEST_DIR");
+    let manifest_dir = manifest_dir.as_ref().map(std::path::Path::new).unwrap();
+    let parent = manifest_dir.parent().unwrap();
+
+    assert!(std::env::set_current_dir(parent).is_ok());
+    let git_grep = std::process::Command::new("git")
```

Comment:
I was thinking about running grep externally, in CI, not in a build script. This will be needlessly slow and has the big issue that it won't work on Windows.

And if people really want to do this then instead of `assert_eq`, just pass the vec into for.
Or possibly just write this:

```rust
const MSRV_MINOR: u64 = 48;
for version in MSRV_MINOR..=minor {
    println!("cargo:rustc-cfg=rust_v_1_{}", version);
}
```

This would produce a bunch of unused cfgs but the compiler doesn't care.

Reply:
> I was thinking about running grep externally, in CI, not in a build script. This will be needlessly slow and has the big issue that it won't work on Windows.

Yeah maybe this is better suited to a CI build script since it will slow down each build a little bit during development.  Why would this not work on windows though?

Reply:
Someone might not have `grep` installed (or any other alternative dependency).

Reply:
One would think in a dev environment grep would be installed, but fail point.